ci/travis_install_toolchain.sh

```diff
@@ -25,7 +25,7 @@ if [ ! -e $CPP_TOOLCHAIN ]; then
     CONDA_PACKAGES=""
 
     if [ $ARROW_TRAVIS_GANDIVA == "1" ] && [ $TRAVIS_OS_NAME == "osx" ]; then
-        CONDA_PACKAGES="$CONDA_PACKAGES llvmdev=6.0.1"
+        CONDA_PACKAGES="$CONDA_PACKAGES llvmdev=$CONDA_LLVM_VERSION"
```

Comment:
Can use the syntax `llvmdev="7.0.*"` instead of hardcoding the bugfix level.

Reply:
done